src/coreclr/.nuget/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.pkgproj

```diff
@@ -0,0 +1,27 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <PackageDescription>ILCompiler.Reflection.ReadyToRun.dll</PackageDescription>
+    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
+    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
```

Comment:
Why do we want to version this package differently?

Reply:
I wanted to use the prerelease versioning to signal the fact that this binaries API surface is subjected to change. Check out https://github.com/dotnet/runtime/issues/41379 for more context about this package.

Reply:
do we intend to always ship this package with the runtime release ? if yes, we should probably use the same converntion i.e. preview label and iteration as 4

Reply:
Chatted offline with @Anipik, it sounded like the right thing to do is to version with the runtime. Fixed.